lib/src/impl_js/impl_js.utils.dart

```diff
@@ -99,6 +99,13 @@ Object _translateDomException(
       '"${e.name}", message: $message');
 }
 
+Object _translateJavaScriptException(Error e) {
```

Comment:
Let's give this a documentation comment.

Let's link to the issue in the SDK, that way when we discover that the issue is resolved, we can work to fix this hack.